docs/scaffold-a-new-project/README.md

```diff
@@ -0,0 +1,72 @@
+| Application | Node.js          | [`neutrino-preset-node`](https://neutrino.js.org/presets/neutrino-preset-node/)                                    |
+| Library     | Web              | [`neutrino-preset-taskcluster-web-library`](https://www.npmjs.com/package/neutrino-preset-taskcluster-web-library) |
+| Components  | React Components | [`neutrino-preset-react-components`](https://www.npmjs.com/package/neutrino-preset-react-components)               |
+
+## Test Runners
+Testing is necessary because we all make mistakes. `create-neutrino-project` offers the following test runners:
+
+| Test Runner | Preset                                                                            |
+|-------------|-----------------------------------------------------------------------------------|
+| Jest        | [`neutrino-preset-jest`](https://neutrino.js.org/presets/neutrino-preset-jest/)   |
+| Karma       | [`neutrino-preset-karma`](https://neutrino.js.org/presets/neutrino-preset-karma/) |
+| Mocha       | [`neutrino-preset-mocha`](https://neutrino.js.org/presets/neutrino-preset-mocha/) |
+
+Be sure to check out the test runner preset to get more information on its features and how files should be named.
+
+## Linters
+Linting is the easiest thing you can do to improve a project. The CLI helper offers two linter choices.
+
+| Linter      | Preset                                                                                                                                                                                                                |
+|-------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Airbnb      | React project: [`neutrino-preset-airbnb`](https://www.npmjs.com/package/neutrino-preset-airbnb) <br> Non React project: [`neutrino-preset-airbnb-base`](https://neutrino.js.org/presets/neutrino-preset-airbnb-base/) |
+| StandardJS  | [`neutrino-middleware-standardjs`](https://www.npmjs.com/package/neutrino-middleware-standardjs)                                                                                                                      |
+
+## Project Layout
+`create-neutrino-project` follows the standard [project layout](https://neutrino.js.org/project-layout.html) specified by Neutrino. This
+means that by default all project source code should live in a directory named `src` in the root of the
+project. This includes JavaScript files, CSS stylesheets, images, and any other assets that would be available
+to import your compiled project.
+
+With it comes to test runners, all project test code should live in a directory named `test` in the
+root of the project. Be sure to check out the test runner preset to get more information on its
+features and how files should be named.
+
```

Comment:
Add a small section about customization, about how you can still customize and configure the project after it has been set up. This can probably be copied from somewhere else. In it you can link to the customization docs.